rerun_py/src/remote.rs

```diff
@@ -54,7 +54,7 @@ impl PyConnection {
     /// List all recordings registered with the node.
     fn list_recordings(&mut self) -> PyResult<Vec<PyRecordingMetadata>> {
         self.runtime.block_on(async {
-            let request = ListRecordingsRequest {};
+            let request = ListRecordingsRequest { columns: None };
```

Comment:
The usage of None here is unfortunate. Maybe we should call this `column_projection`?

Reply:
I went back and forth between "list of strings, empty list means everything" and "optional selection, none mean everything", I thought the latter would be more explicit. I really like ``column_project`` and how explicit it becomes with it! Will go with that if you don't object